joomla/joomla-cms
# [com_redirect] links view: searchtools ordering, bug corrections and some improvements

#### Description

This PR adds, for consistency, as it exists in so many other views in the backoffice, the order select box of the search tools to com_redirect links view.

Also:
- moves the system messages to the view (removes then from the template)
- fixes several bugs in searchtools
- correct visual on mobile, tablet and desktop
- other small graphical issues.
###### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/14602689/31d5d868-0561-11e6-86da-4990822cf4da.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/14602664/1276aede-0561-11e6-9f17-79fd80f201a7.png)
#### Testing Instructions
1. Install latest staging and apply this patch
2. Go to Components -> Redirect
3. Check the message with and without redirect plugin enabled, and with or without collect URL enabled.
4. Check the search, filtering, ordering and limits are working fine.
5. Test desktop, tablet and mobile view (ex: reduce browser width).
